src/components/FilterPanel/FilterPanelLabel/FilterPanelLabel.js

```diff
@@ -67,6 +73,7 @@ FilterPanelLabel.defaultProps = {
   countLabel: count => `${count} items`,
   className: undefined,
   countClassName: undefined,
+  title: 'title',
```

Comment:
I would make this one `undefined` as well because a title is not always necessary and it looks strange having a title that says "title"

Reply:
@diego-codes Just noticed this comment... it looks like `title: 'title'` by default both here and in `FilterPanelAccordionItem`, so I will make it `undefined` in both places 👍

Reply:
That's a good question, I don't remember to be honest. Maybe because the `FilterPanelAccordionItem` must always have a label while `FilterPanelAccordion` doesn't? But I think that making it consistent with the rest might be preferred.